modules/file_upload/cms_file_upload_file.e

```diff
@@ -9,26 +9,35 @@ class
 
 inherit
```

Comment:
I still find this class not wanted, or at least it should not inherit from WSF_UPLOADED_FILE.

WSF_UPLOADED_FILE is in the context of http request ... i.e WSF_REQUEST.